docs/ig.md

```diff
@@ -137,3 +139,21 @@ $ sudo ig list-containers -o json --containername etcd
   }
 ]
 ```
+
+For example, with `--host`, you can get the following output:
+
+```bash
+$ sudo ig trace exec --host
+CONTAINER                                               PID        PPID       COMM             RET ARGS
+
+# Open another terminal.
+$ ls > /dev/null
+$ docker run --name test-host --rm -t debian sh -c 'ls > /dev/null'
+# Go back to first terminal.
+CONTAINER                                               PID        PPID       COMM             RET ARGS
+                               5671             4040             ls               0   /usr/bin/ls --color=auto
+test-host                      5757             5735             sh               0   /bin/sh -c ls > /dev/null
+test-host                      5778             5757             ls               0   /bin/ls
+```
+
+As you can see, events generated from containers have their container field set, while events which are generated from root mount namespace do not.
```

Comment:
```suggestion
As you can see, events generated from containers have their container field set, while events which are generated from the host do not.
```

I would avoid saying "root mount namespace" because that's also true for other mount namespaces as long as they are not from containers.